test/pipeline_test_runner.cc

```diff
@@ -425,19 +430,24 @@ TEST_CASE("PerPhaseTest") { // NOLINT
             for (auto file : files) {
                 core::UnfreezeNameTable nameTableAccess(*rbiGenGs); // enters original strings
 
-                auto settings =
-                    parser::Parser::Settings{false, false, false, gs->cacheSensitiveOptions.rbsSignaturesEnabled};
+                auto collectComments = rbiGenGs->cacheSensitiveOptions.rbsSignaturesEnabled ||
```

Comment:
Can we change the gs to have only one `cacheSensitiveOptions.rbsEnabled` that gets turned on if either one of the two option is on?

Reply:
Opened https://github.com/Shopify/sorbet/pull/431, lmk what you think.